src/test/java/com/jcabi/log/VerboseProcessTest.java

```diff
@@ -477,4 +557,42 @@ public int decide(final LoggingEvent event) {
             return decision;
         }
     }
+
+    /**
+     * Logger appender that compiles a list of all LoggingEvents.
+     */
+    private class TestAppender extends AppenderSkeleton {
+        /**
+         * List of logging events.
+         */
+        private final List<LoggingEvent> logs = new ArrayList<LoggingEvent>(10);
+
+        /**
+         * Provides all captured logging events.
+         * @return Copy of log list
+         */
+        public final List<LoggingEvent> getLogs() {
```

Comment:
@dean-e-clark we cannot have any getters in code: http://www.yegor256.com/2014/09/16/getters-and-setters-are-evil.html